src/main/java/org/mongen/core/controller/OrganizationController.java

```diff
@@ -72,13 +72,15 @@
 	}
 	
 	@ApiOperation(value = "Update a Organization")
+	@CrossOrigin(origins = "*", allowedHeaders = "*")
 	@PatchMapping("/organization/{id}")
 	public ResponseEntity<Organization> updateOrganization(@PathVariable("id") Long id, @RequestBody Organization org){
 		Organization bene = organizationServ.updateOrganization(org, id);
 		return ResponseEntity.status(HttpStatus.OK).body(bene);
 	}
 
 	@ApiOperation(value = "Update status of an Organization")
+	@CrossOrigin(origins = "*", allowedHeaders = "*")
```

Comment:
I'm really surprised that we need to specify this here, as the main CORS config in the main class should apply for all the endpoints, but I'm happy this is working 👍

Reply:
I don't know why tbh. I also googled, that the way you did, should enable cors for the whole app...
I agree we shouldn't put stuff in the crutches in our code, buuuut